docs/src/pages/[platform]/components/shared/GetFieldControls.tsx

```diff
@@ -19,29 +20,33 @@ export interface GetFieldControlsProps {
 export const GetFieldControls = ({ fields }: GetFieldControlsProps) => {
   return (
     <Flex direction="column">
-      {fields.map(([value, setter, name, type = 'text']) =>
-        type === 'checkbox' ? (
-          <CheckboxField
-            name={name}
-            value={value as string}
-            checked={Boolean(value)}
-            label={name}
-            onChange={(event) => {
-              setter(event.target.checked);
-            }}
-          />
-        ) : (
-          <TextField
-            name={name}
-            placeholder={`Set ${name}`}
-            value={value}
-            label={name}
-            onChange={(event: any) => {
-              setter(event.target.value);
-            }}
-          />
-        )
-      )}
+      {fields
+        .map(addKey)
```

Comment:
this line is new

Reply:
Will these get recreated on every render? If so we may as well just use `index` to keep this simple

Reply:
alright...revert it back to use name+index, which I think is good enough